Modules.md

```diff
@@ -175,12 +175,12 @@ to allow *explicitly* sharing state between module instances.
 ## Module start function
 
 If the module has a start node defined, the function it refers should be called
-by the loader after the instance is initialized and before the exported functions
-are called.
+by the loader after the instance is initialized, including its Data and Element
```

Comment:
Wording nit: sections don't get initialised, memories and tables do (from them).